src/main.js

```diff
@@ -1,6 +1,11 @@
+import "vuetify/styles";
 import { createApp } from "vue";
 import App from "./App.vue";
 import router from "./router";
 import store from "./store";
+import vuetify from "./plugins/vuetify";
+import { loadFonts } from "./plugins/webfontloader";
```

Comment:
Should this be `import loadFonts from "./plugins/webfontloader";` ?

Reply:
I copy/pasted from vuetify docs but I can switch it and I am sure it'd work.

Reply:
Nevermind loadFonts is not the default export from webfontloader. The build succeeds when I change this but the homepage doesn't render. Something to keep in mind going forward though?